main.go

```diff
@@ -72,6 +77,8 @@ const (
 	scorecardPolicyFile = "./policy.yml"
 	scorecardFork       = "SCORECARD_IS_FORK"
 	sarif               = "sarif"
+	fulcioURL           = "https://fulcio.sigstore.dev"
```

Comment:
maybe remove this and use
https://github.com/sigstore/cosign/blob/d2781b81e9d0bb6149aec4dd99bffee9fe197fc8/cmd/cosign/cli/options/rekor.go#L22
and 
https://github.com/sigstore/cosign/blob/d2781b81e9d0bb6149aec4dd99bffee9fe197fc8/cmd/cosign/cli/options/fulcio.go#L22